src/test/prover_test.cpp

```diff
@@ -516,14 +548,18 @@ bool TestInvalidJS2In2(
     bits384 trap_r_bits384 = get_bits384_from_vector(hexadecimal_str_to_binary_vector("0F000000000000FF00000000000000FF00000000000000FF00000000000000FF00000000000000FF00000000000000FF"));
     bits256 a_sk_bits256 = get_bits256_from_vector(hexadecimal_digest_to_binary_vector("FF0000000000000000000000000000000000000000000000000000000000000F"));
     bits256 rho_bits256 = get_bits256_from_vector(hexadecimal_digest_to_binary_vector("FFFF000000000000000000000000000000000000000000000000000000009009"));
-    bits256 a_pk_bits256 = get_bits256_from_vector(hexadecimal_digest_to_binary_vector("6461f753bfe21ba2219ced74875b8dbd8c114c3c79d7e41306dd82118de1895b"));
-    bits256 nf_bits256 = get_bits256_from_vector(hexadecimal_digest_to_binary_vector("69f12603c2cfb2acf6f80a8f72cbdeb4417a6b8c7290e793c4d22830c4b35c5f"));
-
-    // We compute a commitment to insert it in the merkle tree we use for the tests
-    // Converted from old hex string "823d19485c94f74b4739ba7d17e4b434693086a996fa2e8d1438a91b1c220331" (big-endian)
-    FieldT cm_field = FieldT("58908622481300953619931625205032657328696563920286427818865722362743092282161");
-    libff::bit_vector address_bits = {1, 0, 0, 0}; // 4 being the value of ZETH_MERKLE_TREE_DEPTH_TEST
+    bits256 a_pk_bits256 = get_bits256_from_vector(hexadecimal_digest_to_binary_vector("5c36fea42b82800d74304aa4f875142b421b4f2847e7c41c1077fbbcfd63f886"));
+    bits256 nf_bits256 = get_bits256_from_vector(hexadecimal_digest_to_binary_vector("d7b310c2179ffb1561870e7783ef812f49b86c368ec1688da6973490530ad731"));
+    // bits256 cm_bits256 = get_bits256_from_vector(hexadecimal_digest_to_binary_vector("04b16ce3099aebfe41699a8b9a057a0b3ffb843dbcd31bfe6e7231d2e260601c"));
+    FieldT cm_field = FieldT("2122734834323728180815849592029731992575252764947574250076821657641665191964");
+    libff::bit_vector address_bits = {1, 0, 0, 0}; // 4 being the value of ZETH_MERKLE_TREE_DEPTH
     const size_t address_commitment = 1;
+    bits256 h_sig = get_bits256_from_vector(hexadecimal_digest_to_binary_vector("6838aac4d8247655715d3dfb9b32573da2b7d3360ba89ccdaaa7923bb24c99f7"));
+    bits256 phi = get_bits256_from_vector(hexadecimal_digest_to_binary_vector("403794c0e20e3bf36b820d8f7aef5505e5d1c7ac265d5efbcc3030a74a3f701b"));
+
+    bits256 h0 = get_bits256_from_vector(hexadecimal_digest_to_binary_vector("fd50e7d159159529144311785b554cb76788fc280d3b3bed50d47e31e5f936d9"));
```

Comment:
All these `h0` and `h1` are set but not used (if you run `make check` you should get some warning). Why did you introduce them?

Reply:
I computed them in case we wanted to check them. They are not necessary in the test.